### ISSUE TITLE
Re-enable "fly under" in Phase 1 curriculum

### ISSUE THREAD

**Issue by gabbard** [isi-vista/adam#672]
It was disabled in 879f14d3b32140992821e000d378c5b4c04c778d because learning it was hanging.

**Comment by paynesa** [comment]
https://github.com/isi-vista/adam/blob/307bc8244f65dd908aa9d8afd420f3174805d976/adam/ontology/phase1_ontology.py#L772-L778
The under/over relations that were bidirectional have been updated to have unidirectional counterparts. 

Previously when implementing above/below in just English, since our English language generator counts non-salient objects as well, we would express "X is under Y" with two relations (X under Y), (Y under X) but keep only one object salient so that both relations weren't translated. 

Since the Chinese language generator requires both objects to be salient, this didn't work in Chinese, so instead I created parallel relations for "X is under Y" given by just (X under Y) so that the single relation is translated and both objects are salient. This matters less for VP's since both function well in this context, but the singular relation is the only one that works for NP's currently.
